src/project/qml/MuseScore/Project/NewScoreDialog.qml

```diff
@@ -58,9 +58,17 @@ StyledDialogView {
             Layout.fillHeight: true
             Layout.fillWidth: true
 
+            anchors.top: parent.top
```

Comment:
you shouldn't use anchors for items in Layout. 
please, see https://doc.qt.io/qt-5/qml-qtquick-layouts-columnlayout.html

Reply:
I think anchors are easier here so I'll just remove the ColumnLayout unless there are any objections?

Reply:
I disagree, just compare how it looked before and how it looks now